examples/rh-sso/client-registrations/redis_pool.lua

```diff
@@ -0,0 +1,28 @@
+local redis = require 'resty.redis'
+
+-- Redis connection parameters
+local _M = {
+  host      = os.getenv("REDIS_HOST") or '127.0.0.1',
+  port      = 6379,
```

Comment:
Actually this is something that should be abstracted in APIcast itself. 

There is no reason to duplicate this code.